doc/classes/EditorExportPlugin.xml

```diff
@@ -81,7 +84,7 @@
 			<param index="1" name="type" type="String" />
 			<param index="2" name="features" type="PackedStringArray" />
 			<description>
-				Virtual method to be overridden by the user. Called for each exported file, providing arguments that can be used to identify the file. [param path] is the path of the file, [param type] is the [Resource] represented by the file (e.g. [PackedScene]) and [param features] is the list of features for the export.
+				Virtual method to be overridden by the user. Called for each exported file except for resources that are handled by [method _customize_resource]. The arguments that can be used to identify the file. [param path] is the path of the file, [param type] is the [Resource] represented by the file (e.g. [PackedScene]) and [param features] is the list of features for the export.
```

Comment:
```suggestion
				Virtual method to be overridden by the user. Called for each exported file, except for resources that are handled by [method _customize_resource]. The arguments that can be used to identify the file. [param path] is the path of the file, [param type] is the [Resource] represented by the file (e.g. [PackedScene]) and [param features] is the list of features for the export.
```
Also, wait. If this is not called "for resources that are handled by `_customize_resource`", what about `_customize_scene`?

Reply:
It is, and always has been(?), called for resources that are handled by `_customize_scene`. Maybe this should be explicitly stated as well?

Reply:
I'm sorry, this change I had suggested was wrong: of all the types that are handled by `_customize_resource`, some are also handled by `_export_file` and others are not.

For example, `png` files are handled only by `_customize_resource` and not by `_export_file`. Conversely, `ogv` files are handled by both `_customize_resource` and not by `_export_file`.

As mentioned in #90365, `_export_file` does not handle any "imported resources", which I guess explains this behaviour.

...I guess I'll take another stab at writing this in a way that explains this.

Reply:
I've changed this to wording that I believe is accurate to the `_export_file` behaviour that has existed for a long time.